ocs_ci/ocs/longevity_stage2.py

```diff
@@ -0,0 +1,563 @@
+import logging
+import statistics
+from datetime import datetime, timedelta
+from concurrent.futures import ThreadPoolExecutor
+
+from ocs_ci.helpers import helpers, performance_lib
+from ocs_ci.ocs import constants, scale_noobaa_lib, ocp
+from ocs_ci.ocs.bucket_utils import (
+    compare_bucket_object_list,
+    sync_object_directory,
+    wait_for_cache,
+)
+from ocs_ci.ocs.ocp import switch_to_project
+from ocs_ci.ocs.resources.pvc import delete_pvcs
+from ocs_ci.ocs.resources.pod import delete_pods
+import ocs_ci.ocs.exceptions as ex
+
+logger = logging.getLogger(__name__)
+
+
+def write_empty_files_to_bucket(
+    mcg_obj, awscli_pod_session, bucket_name, test_directory_setup
+):
+    """
+    Write empty files to bucket and verify if they are created.
+
+    Args:
+        mcg_obj (MCG) : An MCG object containing the MCG S3 connection credentials
+        awscli_pod_session : Fixture to create a new AWSCLI pod for relaying commands.
+        bucket_name (str) : Name of the bucket on which files are to be written.
+        test_directory_setup : Fixture to setup test DIRs.
+
+    Raises:
+        UnexpectedBehaviour : Raises an exception if files are not created.
+
+    Returns:
+        Set: A set of names of all bucket objects.
+
+    """
+
+    full_object_path = f"s3://{bucket_name}"
+    data_dir = test_directory_setup.origin_dir
+
+    # Touch create 1000 empty files in bucket
+    command = f"for file_no in $(seq 1 1000); do touch {data_dir}/test$file_no; done"
+    awscli_pod_session.exec_sh_cmd_on_pod(command=command, sh="sh")
+    # Write all empty objects to the bucket
+    sync_object_directory(awscli_pod_session, data_dir, full_object_path, mcg_obj)
+
+    logger.info("Successfully created files.")
+
+    obj_set = set(obj.key for obj in mcg_obj.s3_list_all_objects_in_bucket(bucket_name))
+    test_set = set("test" + str(file_no + 1) for file_no in range(1000))
+
+    if test_set != obj_set:
+        raise ex.UnexpectedBehaviour("File name set does not match")
+    logger.info("File name set match")
+
+    return obj_set
+
+
+def measure_attach_time(pod_objs, pvc_size):
```

Comment:
why do we need pvc_size for measuring pvc attach time to a pod?

Reply:
No, we do not need it. I'll remove it.